src/Data/Skills/sup_dex.lua

```diff
@@ -3419,7 +3422,10 @@ skills["SupportSlowerProjectiles"] = {
 		},
 		["projectiles_damage_+%_to_nearby_targets"] = {
 			mod("Damage", "INC", nil, ModFlag.Projectile)
-		}
+		},
```

Comment:
This appears to be the same stat as the previous one in the statMap, the wording just got changed.  I also don't think it should be dependent on nearby enemies, as most users won't realize that is what needs to change to make the damage visible.